### DISCUSSION TITLE
TermOpen autocommand does not trigger

### DISCUSSION THREAD

**Discussion by linusboehm** [folke/snacks.nvim#48] [Q&A] [answered]
I'm trying to run an autocommand on TermOpen, but it doesn't seem to trigger. I tried a couple of different approaches, e.g.:
`vim.cmd("autocmd! TermOpen * <command>")`

**Comment by folke** [comment, marked as answer]
This should work:

```lua
vim.api.nvim_create_autocmd("TermOpen", {
  callback = function(ev)
    Snacks.notify("Terminal opened!")
  end,
})
```

In case of toggling a terminal, `TermOpen` is only called on the first open.
